docs/content/mongocxx-v3/installation/macos.md

```diff
@@ -89,7 +61,7 @@ cd mongo-cxx-driver-r3.8.0/build
 Make sure you change to the `build` directory of whatever source tree you
 obtain.
 
-### Step 4: Configure the driver
+### Step 3: Configure the driver
 
 On Unix systems, `libmongoc` installs into `/usr/local` by default. Without additional
```

Comment:
```suggestion
Without additional
```

Suggest removing mention of `libmongoc` install path since users following this tutorial will not need to install the C driver manually.